kickstart/provision_rhel.erb

```diff
@@ -42,7 +42,12 @@ skipx
 <% else -%>
   <% dhcp = !@static -%>
 <% end -%>
+
+<% if rhel_compatible && os_major == 5 -%>
```

Comment:
This file's for RHEL only, so it doesn't define rhel_compatible.  Just having the os_major check should suffice.